src/domain/app/layout/PageWrapper.tsx

```diff
@@ -12,12 +13,21 @@ interface Props {
 const PageWrapper: React.FC<Props> = ({ children, title = 'appName' }) => {
   const { t } = useTranslation();
   const locale = useLocale();
+  const { trackPageView } = useMatomo();
   const { asPath: pathname } = useRouter();
+
   const translatedTitle =
     title !== 'appName'
       ? `${t(title)} - ${t('common:appName')}`
       : t('common:appName');
 
+  // Track page changes when pathnname changes
```

Comment:
What is the level of analytics we try to achieve? As far as I know, we could track that does users follow certain links, do interactions, how long they stay on the site etc.

Reply:
I have no idea what kinds of analytics we want.. I think the task was to get this set up. I haven't investogated in matamo what is it doing by default.

Reply:
This page change was basically copied from events-helsinki-ui

Reply:
Requirements for analytics should come from stakeholders